<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/new/icons/icon</normaloff>:/new/icons/icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeViewWorkspace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>779</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkspace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Workspace:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkspacePath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPato_SCM">
    <property name="title">
     <string>Pato-SCM</string>
    </property>
    <addaction name="actionEnvironment_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionCheckin"/>
    <addaction name="actionCheckout"/>
   </widget>
   <addaction name="menuPato_SCM"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCheckout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/new/icons/checkouticon</normaloff>:/new/icons/checkouticon</iconset>
   </property>
   <property name="text">
    <string>Check-out</string>
   </property>
  </action>
  <action name="actionCheckin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/new/icons/checkinincon</normaloff>:/new/icons/checkinincon</iconset>
   </property>
   <property name="text">
    <string>Check-in</string>
   </property>
  </action>
  <action name="actionEnvironment_Settings">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/new/icons/settingsicon</normaloff>:/new/icons/settingsicon</iconset>
   </property>
   <property name="text">
    <string>Environment Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
